lib/bundle_update_interactive/cli/multi_select.rb

```diff
@@ -51,6 +53,18 @@ def prompt
 
     attr_reader :pastel, :table, :tty_prompt, :title
 
+    def add_keybindings
+      tty_prompt.on(:keypress) do |event|
+        if %w[k p].include?(event.value)
+          tty_prompt.trigger(:keyup)
+        end
+
+        if %w[j n].include?(event.value)
+          tty_prompt.trigger(:keydown)
+        end
+      end
```

Comment:
Hmm, GitHub lost my review comment...? Let me try again.

```suggestion
      tty_prompt.on(:keypress) do |event|
        tty_prompt.trigger(:keyup) if %w[k p].include?(event.value)
        tty_prompt.trigger(:keydown) if %w[j n].include?(event.value)
      end
```

Reply:
Ah--sorry I didn't check rubocop. I will fix when I'm back at my computer.

Reply:
OK, resolved :)

Thanks, Matt!